samples/app-anonymous-users/nodejs/README.md

```diff
@@ -0,0 +1,292 @@
+---
+page_type: sample
+description: This sample shows anonymous users support in meeting apps.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "22-02-2023 10:00:01"
+urlFragment: officedev-microsoft-teams-samples-app-anonymous-users-nodejs
+
+---
+
+## Anonymous User Support
+
+This sample shows anonymous users support in meeting apps.
```

Comment:
This sample shows anonymous users support in `Teams` meeting apps.

Reply:
Fixed